### PR TITLE
Disable add attributes to user store when the user store is disabled

### PR DESCRIPTION
**Pull request by dasuni-30**
### Related Issues
- None

### Related PRs
- None

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/claims/components/add/add-local-claim.tsx [hunk 1]
```diff
@@ -101,21 +102,52 @@ export const AddLocalClaims: FunctionComponent<AddLocalClaimsPropsInterface> = (
...
+        if (attributeConfig.localAttributes.createWizard.showPrimaryUserStore) {
+            userstore.push({
+                description: "",
+                enabled: true,
+                id: "PRIMARY",
+                name: "PRIMARY",
+                self: ""
+            });
+        }
+        getUserStoreList().then((response) => {
```
Comment (brionmario) [inline apps/console/src/features/claims/components/add/add-local-claim.tsx:119]:
```suggestion
        
        getUserStoreList().then((response) => {
```

File: apps/console/src/features/claims/components/add/add-local-claim.tsx [hunk 2]
```diff
@@ -101,21 +102,52 @@ export const AddLocalClaims: FunctionComponent<AddLocalClaimsPropsInterface> = (
     const [ alert, setAlert, alertComponent ] = useWizardAlert();
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
+    const [ userStore, setUserStore ] = useState<UserStoreListItem[]>([]);
+
+    useEffect(() => {
+        const userstore: UserStoreListItem[] = [];
+
+        if (attributeConfig.localAttributes.createWizard.showPrimaryUserStore) {
```
Comment (brionmario) [inline apps/console/src/features/claims/components/add/add-local-claim.tsx:110]: Let's add null checks.

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/claims/components/wizard/local-claim/mapped-attributes.tsx [hunk 3]
```diff
@@ -125,24 +125,29 @@ export const MappedAttributes: FunctionComponent<MappedAttributesPropsInterface>
...
+                                                        requiredErrorMessage={ t("console:manage.features.claims." +
+                                                            "local.forms.attribute.requiredErrorMessage") }
+                                                        value={ values?.get(store.name)?.toString() }
+                                                        data-testid={ `${ testId }-form-store-name-input` }
+                                                    />
+                                                </Grid.Column>
+                                            </Grid.Row>
+                                        ) }
+                                    </>
+
```
Comment (inthirakumaaran) [inline apps/console/src/features/claims/components/wizard/local-claim/mapped-attributes.tsx:150]: remove this line

File: apps/console/src/features/claims/components/add/add-local-claim.tsx [hunk 4]
```diff
@@ -102,53 +102,31 @@ export const AddLocalClaims: FunctionComponent<AddLocalClaimsPropsInterface> = (
...
-            }
-        });
 
-    }, [ hiddenUserStores, userStore ]);
+                setShowMapAttributes(state.length > 0 && userStoresEnabled);
+            });
+        } else {
+            setShowMapAttributes(true);
+        }
+
```
Comment (inthirakumaaran) [inline apps/console/src/features/claims/components/add/add-local-claim.tsx:128]: Remove this line

**Review by inthirakumaaran — APPROVED** [review]

File: apps/console/src/features/claims/components/wizard/local-claim/mapped-attributes.tsx (hunk 3 above)
Comment (dasuni-30) [reply apps/console/src/features/claims/components/wizard/local-claim/mapped-attributes.tsx]: Fixed by f6a0cee2ab55da211fbc3fcf3c6e4dfc694f3c9c.

File: apps/console/src/features/claims/components/add/add-local-claim.tsx (hunk 4 above)
Comment (dasuni-30) [reply apps/console/src/features/claims/components/add/add-local-claim.tsx]: Fixed by f6a0cee2ab55da211fbc3fcf3c6e4dfc694f3c9c.

**Pull request merged**